localstack/utils/tail.py

```diff
@@ -65,7 +65,10 @@ def _log_listener(line, *args, **kwargs):
             raise FileNotFoundError
 
         return ShellCommandThread(
-            cmd=["tail", "-f", self.file_path], quiet=False, log_listener=_log_listener
+            cmd=["tail", "-f", self.file_path],
+            quiet=False,
+            log_listener=_log_listener,
```

Comment:
I guess it isn't necessary, but it might also come in handy to have the filename here as well in the thread name?